src/modapi/moditem.h

```diff
@@ -72,4 +73,56 @@ struct CModAPI_ModItem_LineStyle
 	int m_AnimationSpeed;
 };
 
+enum ESkinModifier
```

Comment:
Don't use name for enum, TW don't use it.

Each enum must start with MODAPI_.
PART_KEEP -> MODAPI_SKINMODFLAG_KEEP.

The SKIN_ enums are use for what ?